src/pages/Home.tsx

```diff
@@ -1,13 +1,22 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import Map from '../components/Map/Map';
 import {Text, Heading, Box, Flex, Spacer} from '@chakra-ui/react';
+import AQIDial from '../components/AQIDial';
 
 /**
  * Home screen component
  */
 const Home: () => JSX.Element = () => {
-  // State for which sensor to display in the current sensor box
+  // --------------- State maintenance variables ------------------------
   const [currentSensor, setCurrentSensor] = useState('');
+  const [displayDial, setDisplayDial] = useState(false);
```

Comment:
My apologies for not catching this earlier, but I think having this state variable and the useEffect are repetitive here. Since the empty string is a falsey value, it already doubles as a displayDial variable

So: remove the useEffect and the displayDial state, and I'll add a comment about how to update the code below